commit 71573aba3535c93478e3c8a3ff8d0fa2096372ec (refs/changes/30/4730/1)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2018-08-21T23:03:50+00:00 (1 year, 2 months ago)
    
    MORAY-490 Electric Moray should pass through UpdateBucket and DeleteBucket options

diff --git a/lib/server.js b/lib/server.js
index c019a11..18deb9a 100644
--- a/lib/server.js
+++ b/lib/server.js
@@ -573,7 +573,7 @@ function updateBucket(options) {
         }
 
         options.clients.array.forEach(function (client) {
-            client.updateBucket(name, cfg, function (err2) {
+            client.updateBucket(name, cfg, opts, function (err2) {
                 log.debug({
                     err: err2
                 }, 'updateBucket: returned');
@@ -647,7 +647,7 @@ function delBucket(options) {
         }
 
         options.clients.array.forEach(function (client) {
-            client.delBucket(name, function (err2) {
+            client.delBucket(name, opts, function (err2) {
                 log.debug({
                     err: err2
                 }, 'delBucket: returned');
